Marlin/Marlin_main.cpp

```diff
@@ -314,7 +314,7 @@ bool target_direction;
   };
 #endif
 
-#ifdef SERVO_ENDSTOPS
+#if ENABLED(HAS_SERVO_ENDSTOPS)
```

Comment:
But, if it _was_ set to `true` this line could simply be `#if HAS_SERVO_ENDSTOPS`, which is how all the other `HAS_*` defines work.